etc/layernames.pegjs

```diff
@@ -110,6 +112,9 @@ spec "Layer specification"
             name: layername.trim()
         };
     }
+    / _ {
```

Comment:
This allows _ (whitespace) to be a valid "spec" or subsection of a layer name. so that " +foo" will break down into " " "+" and "foo"